File: java/src/questao_10/Main.java
package questao_10;

import java.util.Scanner;

public class Main {

    public static boolean isPrimeNumber(int number) {
        if (number <= 1) {
            return false;
        }
        
        for (int i = 2; i < number; i++) {
            if (number % i == 0) {
                return false;
            }
        }
        
        return true;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Digite um numero:");
        int number = scanner.nextInt();
        scanner.close();
        System.out.println("O numero " + (isPrimeNumber(number) ? "é primo" : "não é primo"));
    }
}

File: python/src/questao_10/main.py
def isPrimeNumber(number:int):
    if(number <= 1):
        return False
    
    for i in range(2,number):
        if number % i == 0:
             return False
 
        return True


def main():
   print("Digite um numero:")
   number = int(input())
   print("O numero","é primo" if isPrimeNumber(number) else "não é primo")

if __name__ == '__main__':
    main()